packages/evolution-common/src/services/widgets/WidgetConfig.ts

```diff
@@ -319,6 +319,10 @@ export type QuestionWidgetConfig<CustomSurvey, CustomHousehold, CustomHome, Cust
     path: InterviewResponsePath<CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
     containsHtml?: boolean;
     label: I18nData<CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
+    // When the conditional triggers a hide status and then a show status later on,
```

Comment:
If you put this comment inside `/** blabla */`, then the text will be used as documentation for this field. The way you do now is a simple comment in the code. Since you went through the trouble of documenting this value, you might as well put it in `/** */`

Reply:
done